Spigot-API-Patches/0268-Introduce-beacon-activation-deactivation-events.patch

```diff
@@ -0,0 +1,74 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Spyridon Pagkalos <spyridon@ender.gr>
+Date: Mon, 25 Jan 2021 23:24:15 +0200
+Subject: [PATCH] Introduce beacon activation/deactivation events
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/event/block/BeaconActivatedEvent.java b/src/main/java/com/destroystokyo/paper/event/block/BeaconActivatedEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..aea560ab33c9f539b5876914349d79e362db5457
+--- /dev/null
++++ b/src/main/java/com/destroystokyo/paper/event/block/BeaconActivatedEvent.java
+@@ -0,0 +1,28 @@
++package com.destroystokyo.paper.event.block;
++
++import org.bukkit.block.Block;
++import org.bukkit.event.HandlerList;
++import org.bukkit.event.block.BlockEvent;
++import org.jetbrains.annotations.NotNull;
++
++/**
++ * Called when a beacon is activated.
```

Comment:
What is an activation? Setting an effect? When a block is placed to make the beam start? Make it less ambiguous.

Reply:
Added a second line with a description of the activation, take a look!